ping/ping6_common.c

```diff
@@ -372,15 +372,6 @@ int ping6_run(struct ping_rts *rts, int argc, char **argv, struct addrinfo *ai,
 	   )
 		error(2, errno, _("can't receive hop limit"));
 
-	if (rts->opt_tclass) {
-#ifdef IPV6_TCLASS
-		if (setsockopt(sock->fd, IPPROTO_IPV6, IPV6_TCLASS, &rts->tclass, sizeof rts->tclass) == -1)
-			error(2, errno, _("setsockopt(IPV6_TCLASS)"));
-#else
-		error(0, 0, _("traffic class is not supported"));
-#endif
-	}
-
```

Comment:
@gault I wonder if 78aa5c1516d6f299331b4cd0efac5f0f53a28c0f brought the regression or if it was later by some Pavel Simerda changes.

Reply:
I think it's commit ebad35fee3de ("ping: merge `ping6` command into `ping`"). It removed the "options |= F_TCLASS;" statement that was in ping6_main(), but didn't remove the "if (options & F_TCLASS) {" test in ping6_run().

I didn't put any Fixes: tag because that was just dead code elimination. But if we want to point to a commit, I think that should be ebad35fee3de.